RepoDb.Core/RepoDb/IdOptions.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Linq.Expressions;
+
+namespace RepoDb
+{
+    internal class IdOptions<T> : IIdOptions<T> where T : class
```

Comment:
Why is this internal?

Reply:
No need to expose this class to the "world". We expose the interface. Basically if the class itself would "hold" some internal properties like  PorpertyInfo that the framework needs, we won't expose it.